lib/services/api/api_list_object.dart

```diff
@@ -0,0 +1,7 @@
+part of 'api_service.dart';
+
+class ApiListObject<E> {
+  const ApiListObject({required this.items});
+
+  final List<E> items;
```

Comment:
seems like a lot of work 😔  with retrofit or alamofire, you just need to define `List<Survey>` in the return type of an API request 😀

Reply:
This `ApiListObject` may contains the meta data also 
```json
"meta": {
    "page": 1,
    "pages": 10,
    "page_size": 2,
    "records": 20
  }
```
And btw, i think `alamofire` is `dio` in flutter. We don't have any something like this `@GET("users/{user}/repos")` in ios 😁

Reply:
Of course, this is a painful way, but I think it's worthy to try to see how good [retrofit](https://pub.dev/packages/retrofit) can provide me 🙈